setup.py

```diff
@@ -113,7 +113,7 @@ def get_install_requirements(install_requires, choose_install_requires):
     return install_requires
 
 
-EXTRAS_REQUIREMENTS = {"desktop": ["fiftyone-desktop>=0.29,<0.30"]}
+EXTRAS_REQUIREMENTS = {"desktop": ["fiftyone-desktop>=0.30,<0.31"]}
```

Comment:
This was not bumped for `fiftyone==0.22`. Unsure of consequences but definitely seems wrong

Reply:
This definitely results in the `0.29` package being installed

```
root@b1f5e165669a:/# fiftyone migrate --info
Client version: 0.22.1
Compatible versions: >=0.19,<0.22

Database version: 0.22.1
root@b1f5e165669a:/# pip install fiftyone[desktop]
[stuff]
Successfully installed fiftyone-desktop-0.29.0
```